packages/validator/src/lib.js

```diff
@@ -143,37 +154,83 @@ const isSelfIssued = (capability, issuer) => capability.with === issuer
  * @param {API.ValidationOptions<API.ParsedCapability<A, R, API.InferCaveats<C>>>} config
  * @returns {Promise<API.Result<Authorization<API.ParsedCapability<A, R, API.InferCaveats<C>>>, API.Unauthorized>>}
  */
-export const access = async (
-  invocation,
-  { canIssue = isSelfIssued, principal, resolve = unavailable, capability }
+export const access = async (invocation, { capability, ...config }) =>
```

Comment:
I think a human readable js doc explaining when/why to use this could be helpful. In general, I like short names like these packages use, but we also frequently re-use those names across modules/files, and there are a lot of objects whose property names come from the intersection of several types, and those properties sometimes come from exports like this via a glob import.

wdyt of jsdoc human readable explanation here?
wdyt of something like an eslint rule that requires human readable explainers for exported functions?